<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Consultar</class>
 <widget class="QWidget" name="Consultar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="Paises">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>731</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="consultarPaises">
    <attribute name="title">
     <string>Paises</string>
    </attribute>
    <widget class="QPushButton" name="btSalir">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>80</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>81</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PAISES</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtConsultarPaises">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>501</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btConsPaises">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>390</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="modificarTiempo">
    <attribute name="title">
     <string>Ciudades</string>
    </attribute>
    <widget class="QPushButton" name="btCancelarTiempo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>501</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>270</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese el código del país: </string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCodPaisSalida_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>100</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>121</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CIUDADES</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btCancelarPrecio_5">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>390</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Conexiones</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>501</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>199</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese el código del país: </string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCodPaisSalida_3">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>90</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>151</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CONEXIONES</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btCancelarPrecio_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btCancelarPrecio_4">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>380</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="modificarAsientos">
    <attribute name="title">
     <string>Trenes</string>
    </attribute>
    <widget class="QPushButton" name="btSali2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConsT">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>380</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>91</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TRENES</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCodT">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>100</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtConsultarT">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>501</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>270</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese el código del tipo de tren: </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Precio</string>
    </attribute>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>270</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese el código del país: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>270</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese el código de la ciudad: </string>
     </property>
    </widget>
    <widget class="QPushButton" name="btConsPre">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>380</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCiudad">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>110</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtPais">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>70</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>91</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PRECIO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btSalir_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtConsPrec">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>501</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>270</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese el código de la conexión: </string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCone">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>150</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Número de asientos</string>
    </attribute>
    <widget class="QTextEdit" name="txtCodTT">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>90</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="btSalir6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>261</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>NÚMERO DE ASIENTOS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>270</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese el código del tipo de tren: </string>
     </property>
    </widget>
    <widget class="QPushButton" name="btConsNA">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>380</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCNA">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>551</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>270</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese el código del tren: </string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCodT_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>130</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Rutas</string>
    </attribute>
    <widget class="QTextEdit" name="txtCodTTren">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>90</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="btSalir_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>81</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RUTAS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>270</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese el código del tipo de tren: </string>
     </property>
    </widget>
    <widget class="QPushButton" name="btConsR">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>370</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtRutas">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>501</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>270</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese el código del tren: </string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCodTren">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>130</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btSalir</sender>
   <signal>clicked()</signal>
   <receiver>Consultar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCancelarTiempo</sender>
   <signal>clicked()</signal>
   <receiver>Consultar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCancelarPrecio_2</sender>
   <signal>clicked()</signal>
   <receiver>Consultar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btSali2</sender>
   <signal>clicked()</signal>
   <receiver>Consultar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btSalir_3</sender>
   <signal>clicked()</signal>
   <receiver>Consultar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btSalir6</sender>
   <signal>clicked()</signal>
   <receiver>Consultar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btSalir_2</sender>
   <signal>clicked()</signal>
   <receiver>Consultar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
